### DISCUSSION TITLE
Admin Profile UI

### DISCUSSION THREAD

**Discussion by yadrielsamir** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#53] [Ideas]
Admin Profile UI Preview

<img width="329" alt="Screenshot 2024-09-05 at 6 57 24 PM" src="https://github.com/user-attachments/assets/c46a413e-9a1b-4048-8004-d4d82ea1df29">

Any suggestions?

**Comment by yadrielsamir** [comment]
Otra opción
<img width="325" alt="Screenshot 2024-09-05 at 7 16 02 PM" src="https://github.com/user-attachments/assets/24638387-b343-4f3b-95cb-9ae6cbf3b554">

**Comment by joel-alvarado** [comment]
I like the first version a lot, my only gripe is maybe lowering the opacity of the university logo (maybe 30% opacity?). Otherwise LGTM 👍

**Reply by yadrielsamir** [reply]
<img width="378" alt="Screenshot 2024-09-06 at 3 08 06 AM" src="https://github.com/user-attachments/assets/f9244f4b-3bff-40ec-9c71-47f025412a0c">


Better?

**Comment by YanehRuiz** [comment]
I agree on the first option. It's very balanced and sophisticated!
